armada_jupyter/armada.py

```diff
@@ -24,28 +28,68 @@ def create_armada_request(job: Job, client: ArmadaClient):
     )
 
 
-def submit(submission: Submission):
+def submit(submission: Submission, job: Job, client: ArmadaClient) -> str:
     """
     Starts a workflow for jupyter notebook.
```

Comment:
Probably should be "Submits a workflow".  Submitting a worklow to Armada means that it will be eventually started pending capacity.